<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>241</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametri Profilo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: rgb(0, 0, 0);
	color: rgb(191, 191, 191);
}



QLineEdit{
	border: solid;
	border-width: 1px;
	border-color: rgb(255,255,255);
}

QComboBox{
	border: solid;
	border-width: 1px;
	border-color: rgb(255,255,255);
}

QDoubleSpinBox{
	border: solid;
	border-width: 1px;
	border-color: rgb(255, 255, 255);
}

QPushButton {
	text-align: center;
	padding: 0px 00px;
	color: rgb(255, 255, 255);
	background-color: rgb(31, 31, 31); 
	border: solid;
	border-width: 2px; 
	border-radius: 10px; 
	border-color: rgb(127, 127, 127)
}
QPushButton:pressed {
	background-color: rgb(64, 64, 64); 
	border-style: inset
}</string>
  </property>
  <widget class="QLabel" name="startLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Inizio</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="endLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fine</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="stepLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Passo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="stepCB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="currentText">
    <string>1 h</string>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>1 min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15 min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30 min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 giorno</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Inserire i parametri dei profili temporali della rete</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="startDte">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endDte">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="showGroupSeparator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pointsLbl">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NN punti</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancella</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="defaultRB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Default data</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>startDte</tabstop>
  <tabstop>endDte</tabstop>
  <tabstop>stepCB</tabstop>
  <tabstop>confirmBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
